src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -923,21 +924,81 @@ void CompilerDirectivesPrintDCmd::execute(DCmdSource source, TRAPS) {
 
 CompilerDirectivesAddDCmd::CompilerDirectivesAddDCmd(outputStream* output, bool heap) :
                            DCmdWithParser(output, heap),
-  _filename("filename","Name of the directives file", "STRING",true) {
+  _filename("filename", "Name of the directives file", "STRING", true),
+  _refresh("-r", "Refresh affected methods.", "BOOLEAN", false, "false") {
```

Comment:
Nit: The dot is not needed at the end, I think. The same applies to lines: 945, 970 and 987.

Reply:
Thanks, the dots were removed.